examples/plugins-hookable/plugins/first.js

```diff
@@ -0,0 +1,12 @@
+
+export default function ({ app: { $loadOrder, $hookOrder } }, inject, hooks) {
```

Comment:
Maybe we can add `inject` and `hooks` properties to the `inject` function object?

1. Keeping backward support
2. We can use the new syntax of `function(ctx, h)`: `function(..., { inject, hooks })`

Reply:
Isnt using a new syntax a breaking change?

Reply:
If we attach it to inject, no:

```js
// backward compatible object
const h = inject
h.inject = inject
h.hook = hook
// pass h as second arg
```

Reply:
Ghehe, I think @Atinux dislikes settings props on functions like this. He shot down a pr of mine because I did this (pr was about hooks too, having the possibility to run a hook only once and remove it afterwards)

Reply:
Yes. Probably because you also skipped entire comments in ideas channel regarding hooks implementation for plugins ;)